core/src/ctx/context.rs

```diff
@@ -240,6 +197,7 @@ impl MutableContext {
 			index_stores: parent.index_stores.clone(),
 			#[cfg(not(target_arch = "wasm32"))]
 			index_builder: parent.index_builder.clone(),
+			//TODO: is there a reason these shouldn't include kv-surrealcs
```

Comment:
No this should include `kv-surrealcs`, so it should be `#[cfg(storage)]` like all the others!